packages/config/config/devices/0x0371/zwa009.json

```diff
@@ -0,0 +1,210 @@
+// Aeotec ZWA009
+// aerQ Temperature and Humidity Sensor
+{
+  "_approved": true,
+  "manufacturer": "Aeotec Ltd.",
+  "manufacturerId": "0x0371",
+  "label": "ZWA009",
+  "description": "aerQ Temperature and Humidity Sensor",
+  "devices": [
+    {
+      "productType": "0x0002",
+      "productId": "0x0009"
+    }
+  ],
+  "firmwareVersion": {
+    "min": "0.0",
+    "max": "255.255"
+  },
+  "supportsZWavePlus": true,
+  "paramInformation": {
+    "1": {
+      "label": "Minimum Temperature change to report",
+      "description": "This value defines the minimum change of temperature to cause an unsolicited report of humidity to the central controller using Lifeline. If the value is set to 0, there will be no reports sent to the controller, when the temperature changes. However, periodic reports, managed by configuration parameter 4, may still be active. Valid values are 0-100. Unit is 1/10 degree. Default: 20.",
+      "unsigned": true,
+      "valueSize": 1,
+      "minValue": 0,
+      "maxValue": 100,
+      "defaultValue": 20,
+      "readOnly": false,
+      "writeOnly": false,
+      "allowManualEntry": true
+    },
+    "2": {
+      "label": "Minimum humidity change to report",
+      "description": "This value defines the minimum change of humidity to cause an unsolicited report of humidity to the central controller using Lifeline. If the value is set to 0, there will be no reports sent to the controller, when the humidity changes. However, periodic reports, managed by configuration parameter 4, may still be active. Valid values are 0-20. Unit is %. Default: 5.",
+      "unsigned": true,
+      "valueSize": 1,
+      "minValue": 0,
+      "maxValue": 20,
+      "defaultValue": 5,
```

Comment:
unit `%` (or `% rH`) missing

Reply:
Not sure what you mean. It says in the description "Unit is %."